corehq/util/sentry.py

```diff
@@ -92,12 +100,40 @@ def sanitize(self, key, value):
 sanitize_system_passwords = HQSanitzeSystemPasswords()
 
 
+def subtype_error(tb: TracebackType, rate_limit_key: str) -> str:
```

Comment:
I haven't seen this typing syntax before, though now I see there are other places in HQ that use it. Could/should we be using it more often?

Reply:
This is [type hinting](https://docs.python.org/3/library/typing.html). It allows IDEs and other tools to do more static checks. Has no runtime impact.

Reply:
cool, thanks